### PR TITLE
Update to support openwrt

### PR DESCRIPTION
**Pull request by dhylands**

### CODE REVIEW

File: build-adapter.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ fi
 echo "============================================================="
 
 if [ -f "package.json" ]; then
-  if [ "${ADDON_ARCH}" == "linux-arm" ]; then
+  if [[ "${ADDON_ARCH}" =~ "linux-" ]]; then
```
Comment (mrstegeman) [inline build-adapter.sh]: Shouldn't this be:

```bash
  if [[ "${ADDON_ARCH}" =~ "linux-arm" ]]; then
```

Comment (dhylands) [reply build-adapter.sh]: I believe that you are correct.

**Comment by mrstegeman** [comment]
Travis is failing:

```
../build-adapter.sh: line 56: /home/owrt-user/.owrt: No such file or directory
The command "./build-addons.sh" exited with 1.
```

**Comment by mrstegeman** [comment]
```
> epoll@1.0.2 install /build/gpio-adapter/node_modules/epoll
> node-gyp rebuild
make: Entering directory '/build/gpio-adapter/node_modules/epoll/build'
  CXX(target) Release/obj.target/epoll/src/epoll.o
arm-openwrt-linux-g++: error: unrecognized command line option '-m64'
epoll.target.mk:103: recipe for target 'Release/obj.target/epoll/src/epoll.o' failed
make: *** [Release/obj.target/epoll/src/epoll.o] Error 1
make: Leaving directory '/build/gpio-adapter/node_modules/epoll/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/owrt-user/.nvm/versions/node/v8.16.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Linux 4.4.0-101-generic
gyp ERR! command "/home/owrt-user/.nvm/versions/node/v8.16.0/bin/node" "/home/owrt-user/.nvm/versions/node/v8.16.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /build/gpio-adapter/node_modules/epoll
gyp ERR! node -v v8.16.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! epoll@1.0.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the epoll@1.0.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/owrt-user/.npm/_logs/2019-05-09T17_34_51_970Z-debug.log
The command "./build-addons.sh" exited with 1.
```

**Comment by dhylands** [comment]
The -m64 error was because I missed merging the package.sh change for the gpio-adapter.

**Comment by mrstegeman** [comment]
Seems to be working now.

**Pull request merged**